### PR TITLE
Sovol config Updates

### PR DESCRIPTION
**Pull request by ils15**
Update Configuration for Sovol Printers

Insert easy coment line to enable/disable 3D touch

### CODE REVIEW

**Review by Farva42 — CHANGES_REQUESTED** [review]
The original Board addition is not needed as this is covered by the CrealityV22 & CrealityV221 configs. It also looks like you are using an older config file ie 2.0.9.2 vs the current 2.1.x which will cause issues on the existing configs but a current 2.1.x for the new examples.

**Comment by ils15** [comment]
My mistake
I thinked v22 is a silent board
I will update v22 and v221 boards to match with new settings
Thanks

**Comment by Farva42** [comment]
V22 is the stock board with A4988 drivers. V221 is the silent board with TMC2208_STANDALONE.

**Comment by ils15** [comment]
> V22 is the stock board with A4988 drivers. V221 is the silent board with TMC2208_STANDALONE.

Great
Now i'm updated things here
thanks

**Review by ils15 — COMMENTED** [review]
Update with Z Probe Offset and Babystepping Combined

**Review by ils15 — COMMENTED** [review]
Parametrize SV01 BTT SKR Mini E3 V3

File: config/examples/Sovol/SV-01/CrealityV22/Configuration_adv.h [hunk 1]
```diff
@@ -1882,12 +1884,12 @@
 // Specify additional languages for the UI. Default specified by LCD_LANGUAGE.
 //
 #if ANY(DOGLCD, TFT_COLOR_UI, TOUCH_UI_FTDI_EVE, IS_DWIN_MARLINUI)
-  //#define LCD_LANGUAGE_2 fr
-  //#define LCD_LANGUAGE_3 de
+  #define LCD_LANGUAGE_2 pt_br
+  #define LCD_LANGUAGE_3 es
```
Comment (thinkyhead) [inline config/examples/Sovol/SV-01/CrealityV22/Configuration_adv.h]: This seems a bit too personalized.

**Comment by thinkyhead** [comment]
I put in the necessary time to clean up and update this stunning submission.
Please review the commits for changes and verify that nothing got lost in the cleanup.

**Review by ils15 — COMMENTED** [review]
Everything seems to be fine
Thanks @thinkyhead

**Comment by thinkyhead** [comment]
Okay, then. I went ahead and corrected the various extant errors, regressions, and outdated options, bringing this into better parity with the development tree bugfix-2.1.x. This ought to be reviewed by some other SOVOL users before it is merged, since it contains many personally tailored preferences that may not be embraced by other users.

**Pull request merged**